src/app/visual/date-range-picker/date-range-picker-visual.component.scss

```diff
@@ -0,0 +1,3 @@
+.date-range-demo {
+  padding: 10px;
```

Comment:
Instead of this one-off padding, we could just add `class="sky-padding-even-default"` to the surrounding `div`.